### ISSUE TITLE
how would I extract other types of files like notes

### ISSUE THREAD

**Issue by jay51** [joz-k/ios_backup_extractor#5]
How do I extract other types of files like my notes in the notes app on the iPhone

**Comment by joz-k** [comment]
Currently this tool is focused on extracting media files. In general, it should be possible to extract any content from the backup. However, the backup format is not officially documented by Apple, and various content types require reverse engineering.
There are commercial tools that give you more access to different types of content. 
As for the open-source, I can see at least these two projects claiming to extract notes:
- https://github.com/renesugar/readnotes
- https://github.com/renesugar/movenotes

(Mostly a note to myself) The notes in the backup are stored in the backup SQLite file named `4f98687d8ab0d6d1a371110e6b7300f6e465bef2`. In the `ZICCLOUDSYNCINGOBJECT` table, the `ZWIDGETSNIPPET` column seems to contain the text portion of the notes.

In short, there is no support for extracting notes or other content, but I may add more data types in the future.

**Comment by jay51** [comment]
you're right! I looked inside the sqlite backup for `4f98687d8ab0d6d1a371110e6b7300f6e465bef2 ` db and I was able to find the information under `ZICCLOUDSYNCINGOBJECT` table in `ZWIDGETSNIPPET` column
